packages/pn-pa-webapp/src/components/Notifications/NotificationSettingsDrawer.tsx

```diff
@@ -0,0 +1,178 @@
+import { useFormik } from 'formik';
+import { ChangeEvent, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import * as yup from 'yup';
+
+import CloseIcon from '@mui/icons-material/Close';
+import {
+  Box,
+  Drawer,
+  FormControl,
+  FormControlLabel,
+  IconButton,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Typography,
+} from '@mui/material';
+import { CustomDropdown, useIsMobile } from '@pagopa-pn/pn-commons';
+import { LANGUAGES } from '@pagopa-pn/pn-commons/src/utility/costants';
+import { ButtonNaked, LangCode } from '@pagopa/mui-italia';
+
+import { BILINGUALISM_LANGUAGES, NewNotificationLangOther } from '../../models/NewNotification';
+import { setAdditionalLanguages } from '../../redux/auth/actions';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks';
+import { RootState } from '../../redux/store';
+
+const NotificationSettingsDrawer = () => {
+  const dispatch = useAppDispatch();
+  const [openDrawer, setOpenDrawer] = useState(false);
+  const { t, i18n } = useTranslation(['notifiche']);
+  const additionalLanguages = useAppSelector(
+    (state: RootState) => state.userState.additionalLanguages
+  );
+  const isMobile = useIsMobile();
+
+  const toggleDrawer = () => setOpenDrawer(!openDrawer);
+
+  const onCloseDrawer = async () => {
+    formik.handleSubmit();
```

Comment:
maybe we can improve this part. we can avoid unneeded submit when the user doesn't change anything. put a check that formik.dirty is true before submit